metadata-ingestion/source_docs/data_lake.md

```diff
@@ -47,20 +47,21 @@ This connector supports both local files as well as those stored on AWS S3 (whic
 
 - CSV
 - TSV
-- Parquet
 - JSON
+- Parquet
 - Apache Avro
 
+Note that schemas are inferred for CSV and TSV filetypes on the basis of the first 100 rows. JSON file schemas are likewise inferred on the basis of the entire file, which may impact performance. Schemas for Parquet and Avro files are extracted as provided.
```

Comment:
Does this behaviour come from spark? Is it configurable?

Reply:
This is done with Python—should it be configurable? Otherwise, we only get headers

Reply:
It's significantly easier to read in part of a CSV/TSV as opposed to part of a JSON — I've been looking into packages like [ijson](https://pypi.org/project/ijson/) for reading in partial JSONs but I think that if you've got a huge JSON file there's other issues at hand.

Reply:
Understood, please document that.

Reply:
Done!